backend/src/main/java/com/github/devtoju/backend/gaptext/models/GapText.java

```diff
@@ -0,0 +1,13 @@
+package com.github.devtoju.backend.gaptext.models;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record GapText(
+        @NotBlank
+        String value,
+        @Size(min=1, max=10)
```

Comment:
OMG I found something with @NotBlank you don't need @Size with min=1 only if min should be more than 1 then it is necessary (Hauptsache ich konnte was dazu schreiben xD)

Reply:
thanks for the hint. I googled too and found this: https://stackoverflow.com/a/11190066

In case of using this annotation for a int variable @size is wrong.
for numerics you have to use @min and @max.
@min(1)
@max(10)
int number = could be 1 to 10 inc 1 and 10